lib/base_wizard.py

```diff
@@ -186,12 +190,25 @@ def choose_hw_device(self):
                 continue
             devices += list(map(lambda x: (name, x), u))
         if not devices:
-            msg = ''.join([
+            msgs = [
                 _('No hardware device detected.') + '\n',
-                _('To trigger a rescan, press \'Next\'.') + '\n\n',
-                _('If your device is not detected on Windows, go to "Settings", "Devices", "Connected devices", and do "Remove device". Then, plug your device again.') + ' ',
-                _('On Linux, you might have to add a new permission to your udev rules.'),
-            ])
+                _('To trigger a rescan, press \'Next\'.') + '\n\n'
```

Comment:
Can you do me a favor? I actually want to hear from users.  Maybe you can add something like:

"If you this problem persists (and you really do have a hardware device connected), please visit https://github.com/Electron-Cash/Electron-Cash/issues to file an issue report with the developers."

Reply:
Will do

Reply:
Thanks man you're awesome 👍

Reply:
I added this, also made the link clickable and the text selectable.